eng/Subsets.props

```diff
@@ -119,23 +121,23 @@
   </ItemGroup>
 
   <ItemGroup Condition="$(_subsetCategory.Contains('installer')) and $(_subset.Contains('depproj'))">
-    <DepprojProjectToBuild Include="$(InstallerProjectRoot)pkg\projects\**\*.depproj" SignPhase="Binaries" />
+    <DepprojProjectToBuild Include="$(InstallerProjectRoot)pkg\projects\**\*.depproj" SignPhase="Binaries" BuildInParallel="false" />
     <ProjectToBuild Include="@(DepprojProjectToBuild)" />
   </ItemGroup>
 
   <ItemGroup Condition="$(_subsetCategory.Contains('installer')) and $(_subset.Contains('pkgproj'))">
-    <PkgprojProjectToBuild Include="$(InstallerProjectRoot)pkg\projects\**\*.pkgproj" SignPhase="MsiFiles" />
+    <PkgprojProjectToBuild Include="$(InstallerProjectRoot)pkg\projects\**\*.pkgproj" SignPhase="MsiFiles" BuildInParallel="false" />
```

Comment:
Does `BuildInParallel="false"` mean every matched project runs in sequence within the group? Or can these still run in parallel and it only affects overall orchestration?

Reply:
The `BuildInParallel="false"` makes all of the projects run sequentially. We'd need a fix for dotnet/arcade#4436 before we can do something more in-depth.

Reply:
Well, I argue in https://github.com/dotnet/arcade/issues/4436 we don't need any arcade change to do this "properly". I don't object to merging this workaround to get the PR through, though.

Reply:
Theoretically I agree with you, but in practice I spent a whole day on it and couldn't get it to work successfully with coreclr's current build infrastructure.

Reply:
Yeah, if I notice it affecting my dev loop perf I'll file something and maybe give it a try, until then I'm fine just having https://github.com/dotnet/arcade/issues/4436 track this in case arcade can end up helping.